src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Remove ARB from Isolation Mode on Arbitrum"
+author: "karpatkey_TokenLogic_ACI"
+discussions: "https://governance.aave.com/t/arfc-remove-arb-from-isolation-mode-on-arbitrum-market/16703"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xbc5471496bbc2beda343625cee22c34fc9672785112cc5d19a25ca87c5b422c3"
+---
+
+## Summary
+
+This publication proposes disabling Isolation Mode for ARB on Aave v3 Arbitrum.
+
+## Motivation
+
+Since the ARB airdrop, April 2023, the Arbitrum ecosystem has experienced exponential like growth and the ARB liquidity has improved significantly.
+
+With the support of the Risk Service providers, this proposal is to disable Isolation Mode and enable ARB to be used in combination with other assets as collateral.
+
+## Specification:
```

Comment:
This Specification is completely arbitrary, not saying anything of what the proposal does

Reply:
True.
I suppose that's a flaw in the initial proposal itself.

This has already been voted on though, should we change it only here or also in the forum?